src/components/navigation/AuthStackNavigator.tsx

```diff
@@ -11,26 +11,28 @@ const Stack = createStackNavigator();
 function AuthNavigator(): React.ReactElement {
   const { theme } = useThemeContext();
   return (
-    <Stack.Navigator
-      initialRouteName="Login"
-      screenOptions={{
-        headerStyle: {
-          backgroundColor: theme.background,
-          borderBottomWidth: 0,
-          elevation: 0,
-        },
-        headerTintColor: theme.fontColor,
-      }}
-    >
```

Comment:
@JongtaekChoi Why did you remove this? It will remove the dark mode behavior in the header as shown below?

![image](https://user-images.githubusercontent.com/27461460/71303040-a37c9800-23f6-11ea-8d72-e5ffe1e63d66.png)

Reply:
@hyochan Oh it's my mistake. 
When I was finding the cause of error about the header option, I tested if the secreenOptions be removed then the error occurred. And I forgot to revert this.